frontend/src/metabase/query_builder/components/view/ViewFooter.jsx

```diff
@@ -191,14 +191,28 @@ const VizTypeButton = ({ question, result, ...props }) => {
   const icon = visualization && visualization.iconName;
 
   return (
-    <ViewButton medium p={[2, 1]} icon={icon} labelBreakpoint="sm" {...props}>
+    <ViewButton
+      medium
+      p={[2, 1]}
+      icon={icon}
+      labelBreakpoint="sm"
+      data-testid="viz-type-button"
+      {...props}
+    >
       {t`Visualization`}
     </ViewButton>
   );
 };
 
 const VizSettingsButton = ({ ...props }) => (
-  <ViewButton medium p={[2, 1]} icon="gear" labelBreakpoint="sm" {...props}>
+  <ViewButton
+    medium
+    p={[2, 1]}
+    icon="gear"
+    labelBreakpoint="sm"
+    data-testid="viz-settings-button"
+    {...props}
+  >
```

Comment:
Added `data-testid` here, so it's easier to distinguish visualization settings button and the "settings" tab on the right sidebar for SQL filters

![CleanShot 2021-07-13 at 17 14 44@2x](https://user-images.githubusercontent.com/17258145/125467720-982a7e25-f1fe-4d48-80de-609c283f0c14.png)